articles/tutorials/building_2d_games/04_game_library/index.md

```diff
@@ -0,0 +1,126 @@
+3. Enter the command `dotnet sln add ./MonoGameLibrary/` to add the newly created class library project to the *MonoGameSnake.sln* solution file.
+
+## Adding a Reference To The Class Library
+Now that the game library project has been created, a reference to it needs to be added in our game project.  Without adding a reference, our game project will be unaware of anything we add to the class library.  To do this:
+
+### Visual Studio Code
+To add the game library project as a reference to the game project in Visual Studio Code:
+
+1. In the Solution Explorer panel, right-click the *MonoGameSnake* project.
+2. Choose "Add Project Reference" from the context menu.
+3. Choose *MonoGameLibrary" from the available options.
+
+> [!TIP]
+> The Solution Explorer panel in VSCode is provided by the C# Dev Kit extension that was installed in [Chapter 02](../02_getting_started/index.md#install-the-c-dev-kit-extension).  If you do not see this panel, you can open it by
+>
+> 1. Opening the *Command Palette* (View > Command Palette).
+> 2. Enter "Explorer: Focus on Solution Explorer View" and select the command.
+
+### Visual Studio 2022
+To add the game library project as a reference to the game project in Visual Studio 2022:
+
+1. In the Solution Explorer panel, right-click the *MonoGameSnake* project.
+2. Select Add > Project Reference from the context menu.
+3. Check the box for the *MonoGameLibrary* project.
+4. Click Ok.
+
+### dotnet CLI
+To add the game library project as a reference to the game project with the dotnet CLI:
+
+1. Open a new Command Prompt or Terminal window in the same directory as the *MonoGameSnake.csproj* C# project file.
+2. Enter the command `dotnet add reference ../MonoGameLibrary/`.  This will add the *MonoGameLibrary* reference to the *MonoGameSnake* game project.
+
+## Clean Up
+When using the *MonoGame Game Library* project template, the generated project contains file similar to a standard MonoGame game project, including a *dotnet-tools.json* manifest file, a *Content.mgcb* file, and a *Game1.cs* file.  For the purposes of this tutorial, we will not need these.   To clean these up, locate the following in the *MonoGameLibrary* project directory and delete them:
+
+1. The *.config/* directory.
+2. The *Content/* directory
+3. The *Game1.cs* file.
+
+> [!TIP]
+> These files are needed in more advanced scenarios such as creating a central code base for game logic that is referenced by other projects of which each target different platforms such as desktop, mobile, and console.  Creating a project structure of this type is out of scope for this tutorial.  
+> 
+> If you would like more information on this, Simon Jackson has written the article [Going cross-platform with MonoGame](https://darkgenesis.zenithmoon.com/going-cross-platform-with-monogame.html) which covers this in more detail.
+
+## Conclusion
```

Comment:
One suggestion to address an issue I often see with these kinds of chapters.  It introduces it, adds it, but then doesn't show you how to use it.
Granted this is likely covered later, but by then the reader will need to refer back to this chapter to refresh.

Best to add a section where you add a simple class and then refer to that class from the main project.  Use a class you intend to create to save rework.

Additonally, add a note if they get a warning because the class in the library cannot be found in the main project, reminding them they have forgotten to add a reference :D (you would be surprised how often this comes up :P)

Reply:
Maybe also include a screenshot of the final setup in Solution Explorer, so the reader can double check what they see is correct.